frontend/src/lib/components/PropertyFilters/components/PropertyValue.tsx

```diff
@@ -142,7 +143,8 @@ export function PropertyValue({
         allowClear: Boolean(value),
         onKeyDown: (e: React.KeyboardEvent) => {
             if (e.key === 'Escape' && e.target instanceof HTMLElement) {
-                e.target.blur()
+                setInput('')
+                setShouldBlur(true)
```

Comment:
I actually think this is dirtier than what we had before. How about just calling `e.target.blur()` directly here and getting rid of the entire `useEffect` above (lines 117 - 123)? @marcushyett-ph

Reply:
If we do that when you hit escape we will persist the value you were typing rather than clearing the input?